documentation/src/main/doc/modules/connectors/partials/META-INF/connector/smallrye-kafka-incoming.adoc

```diff
@@ -65,4 +65,8 @@ Type: _string_ | false |
 
 Type: _string_ | false | 
 
+| *consumer-rebalance-listener.name* | The name set in javax.inject.Named of a bean that implements io.smallrye.reactive.messaging.kafka.KafkaConsumerRebalanceListener. If set the listener will be applied to the consumer.
```

Comment:
Can use use `javax.inject.Named` and `io.smallrye.reactive.messaging.kafka.KafkaConsumerRebalanceListener` (purely cosmetic comment)

Reply:
The change must be done in the connector class.

Reply:
done